game.py

```diff
@@ -20,7 +21,47 @@ def snowman(snowman_word):
     If the player wins and, 
     'Sorry, you lose! The word was {snowman_word}' if the player loses
     """
-    pass
+
+    print(f"Debug info: {snowman_word}, length of word: {len(snowman_word)}")
+    
+    correct_letter_guess_statuses = build_letter_status_dict(snowman_word)
+
+    wrong_guesses_list = []
+
+    arriving_at_complete_word_guess = False
```

Comment:
Nice job using a flag (a variable that is used to indicate that a condition has been met) to control your while loop 👍

```suggestion
    is_game_finished = False
```

It's common practice to name a variable that references a boolean with is so that other devs know that this is flag variable that can change.